packages/mobile/src/identity/contactMapping.ts

```diff
@@ -259,13 +258,19 @@ function* fetchWalletAddresses(e164Number: string) {
 
 // Returns a list of account addresses for the identifier received.
 export function* lookupAccountAddressesForIdentifier(id: string) {
+  const lostAccounts = yield call(fetchLostAccounts)
```

Comment:
If this call fails the whole thing is failing, that's probably why the e2e test is failing since it doesn't have access to Firebase. Instead of doing that we can have an empty array as default if the firebase call fails.

Reply:
Thanks for the help :)